config/int.go

```diff
@@ -5,92 +5,215 @@ import (
 	"strconv"
 )
 
-// GetInt returns the int representation of the value associated with k.
+// AsInt returns the int representation of the value associated with k or panics
+// if unable to do so.
+func AsInt(b Bucket, k string) int {
+	return int(asInt(b, k, 0, false))
+}
+
+// AsIntDefault returns the int representation of the value associated with k,
+// or the default value v if k is undefined.
+func AsIntDefault(b Bucket, k string, v int) int {
+	return int(asIntDefault(b, k, 0, int64(v), false))
+}
+
+// AsIntP returns the int representation of the value associated with k or panics
+// if unable to do so.
 //
-// If k is undefined, ok is false and err is nil.
+// It panics if the value is not zero or negative.
```

Comment:
I think this should not have the *not*?
```suggestion
// It panics if the value is zero or negative.
```

Reply:
Or flip the `negative` to be `positive` as Danil suggested for the others. Is zero allowed for positive?

Reply:
Nah, `0` is not allowed - I have fixed these.